addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/connector/RFXComSerialConnector.java

```diff
@@ -53,7 +53,7 @@ public void connect(RFXComBridgeConfiguration device)
         serialPort = (SerialPort) commPort;
         serialPort.setSerialPortParams(38400, SerialPort.DATABITS_8, SerialPort.STOPBITS_1, SerialPort.PARITY_NONE);
         serialPort.enableReceiveThreshold(1);
-        serialPort.disableReceiveTimeout();
+        serialPort.enableReceiveTimeout(100); // In ms. Small values mean faster shutdown but more cpu usage.
```

Comment:
As we did discover this one does do what we hoped... http://www.docjava.com/book/cgij/jdoc/gnu/io/RXTXPort.html#enableReceiveTimeout(int)